react/tic-tac-toe/src/app/screens/Game/styles.module.scss

```diff
@@ -1,6 +1,7 @@
 .game {
   display: flex;
   flex-direction: row;
+  padding: 2rem;
```

Comment:
It's nice to use `rem`. However here at Wolox, we rather use `px` since it's easier to comply with the design guidelines provided by designers.